Library/Formula/texinfo.rb

```diff
@@ -28,8 +28,8 @@ def install
     # However, it complains about installing epsf.tex in TEXMF/tex/generic/dvips, so let's do that...
     # This somewhat breaks the homebrew philosophy, I am sorry.
     # Also, we don't depend on tex-live, but this directory only exists if it is installed.
-    if File.exist? "#{HOMEBREW_PREFIX}/share/texmf-dist/" then
-      cp "doc/epsf.tex", "#{HOMEBREW_PREFIX}/share/texmf-dist/tex/generic/dvips/"
+    if File.exist?(HOMEBREW_PREFIX/"share/texmf-dist/")
+      cp "doc/epsf.tex", HOMEBREW_PREFIX/"share/texmf-dist/tex/generic/dvips/"
```

Comment:
Let's remove it. `HOMEBREW_PREFIX/"share/texmf-dist/"` won't be created by any known software. (The path inside in MacTeX is `/usr/local/texlive/<version>/texmf-dist/`). So I guess this part of codes never got run in this 5 years.